docs/source/reference/ai/model-train-sklearn.rst

```diff
@@ -6,21 +6,54 @@ Model Training with Sklearn
 1. Installation
 ---------------
 
-To use the `Sklearn framework <https://scikit-learn.org/stable/>`_, we need to install the extra sklearn dependency in your EvaDB virtual environment.
+To use the `Flaml XGBoost AutoML framework <https://microsoft.github.io/FLAML/docs/Examples/Integrate%20-%20Scikit-learn%20Pipeline/>`_, we need to install the extra Flaml dependency in your EvaDB virtual environment.
```

Comment:
Is this the sklearn documentation? Why XGBoost?

Reply:
Corrected the typos